numpy/core/_add_newdocs.py

```diff
@@ -5558,8 +5569,8 @@ def luf(lamdaexpr, *args, **kwargs):
 
     See Also
     --------
-    packbits : Packs the elements of a binary-valued array into bits in a uint8
-               array.
+    packbits : Packs the elements of a binary-valued array into bits in
```

Comment:
The line renders just fine: https://docs.scipy.org/doc/numpy/reference/generated/numpy.unpackbits.html. The only thing I did here was to wrap to 72 columns since this is a comment string.

Reply:
OK, good to hear it resolves fine.